File: GC.Domain/Gardens/Sectors/GardenSectorBlank.cs
using System;

namespace GC.Domain.Gardens.Sectors
{
    public class GardenSectorBlank
    {
        public Guid? Id { get; set; }
        public Int32? SectorNumber { get; set; }
        public String? ElectricityNumber { get; set; }
        public Double? NumberOfAcres { get; set; }
        public String? CadastralNumber { get; set; }
    }
}

File: GC.WebSpace/ClientApp/domain/gardens/gardenSectors/gardenSectorBlank.ts
import { GardenSector } from "./gardenSector";

export class GardenSectorBlank {
    constructor(
        public id: string | null,
        public sectorNumber: number | null,
        public electricityNumber: string | null,
        public numberOfAcres: number | null,
        public cadastralNumber: string | null
    ) { }

    static create = (sector: GardenSector): GardenSectorBlank => {
        return new GardenSectorBlank(sector.id, sector.sectorNumber, sector.electricityNumber, sector.numberOfAcres, sector.cadastralNumber);
    }

    static empty = (): GardenSectorBlank => {
        return new GardenSectorBlank(null, null, null, null, null);
    }
}
